monero-harness/tests/wallet.rs

```diff
@@ -22,11 +22,10 @@ async fn fund_transfer_and_check_tx_key() {
     let alice_wallet = monero.wallet("alice").unwrap();
     let bob_wallet = monero.wallet("bob").unwrap();
 
-    // fund alice
-    monero
-        .init(vec![("alice", fund_alice), ("bob", fund_bob)])
-        .await
-        .unwrap();
+    monero.init_miner().await.unwrap();
+    monero.init_wallet("alice", vec![fund_alice]).await.unwrap();
+    monero.init_wallet("bob", vec![fund_bob]).await.unwrap();
+    monero.start_miner().await.unwrap();
```

Comment:
If we do that we might run into errors when generating blocks with the miner because we bulk-generate blocks when creating alice/bob wallet. It *might* work to just init and start the miner, but I would like to avoid errors that potentially cause side effects as well so I only start the miner *after* bulk generation of blocks i over.